### PR TITLE
retry fix for #259, bump version

### PR DESCRIPTION
**Pull request by jasonbrooks**

### CODE REVIEW

File: build_stage1.sh [hunk 1]
```diff
@@ -62,6 +62,10 @@ fi
 
 ostree remote add --repo=/srv/repo centos-atomic-host --set=gpg-verify=false http://mirror.centos.org/centos/7/atomic/x86_64/repo && ostree pull --depth=-1 --repo=/srv/repo --mirror centos-atomic-host centos-atomic-host/7/x86_64/standard
 
+# temp fix for issue 259
+
+ostree --repo=/srv/repo static-delta generate --from 841fae51e5b68716a9996ddbdb4e543855bbfab9c6e4cb433267b24e41e8bbc1 --to 91fe03fef75652f68a9974261b391faaeb5bd20f33abf09bb1d45511ba2df04e && ostree --repo=/srv/repo summary -u
```
Comment (cgwalters) [inline build_stage1.sh]: If the ostree version is new enough I'd add `--if-not-exists`.

**Comment by cgwalters** [comment]
Seems sane offhand to me, but I didn't verify the commit hashes.

File: build_stage1.sh (hunk 1 above)
Comment (jasonbrooks) [inline build_stage1.sh]: Looks like it's 2017.3-2.el7 -- is that new enough?

Comment (cgwalters) [inline build_stage1.sh]: Yep, definitely.  https://github.com/ostreedev/ostree/pull/375

**Pull request merged**